src/main/java/io/codelex/classesandobjects/practice/LitersPerKilometer.java

```diff
@@ -0,0 +1,47 @@
+package io.codelex.classesandobjects.practice;
+
+import java.util.Scanner;
+
+public class LitersPerKilometer {
+    public static void main(String[] args) {
+//        Scanner scan = new Scanner(System.in);
```

Comment:
Kāpēc šis palicis aizkomentēts?

Reply:
Exercise 2 uzdevuma sākumā bija jāizveido Car klase ar startKilometers un endKilometers ... konstruktorā un LitersPerKilometer, lai to notestētu, bet uzdevumā to Car klasi bija jāmaina, lai ņem tikai odometra vērtību. Jauno Car klasi testēju ar CarTest (bija prasīts notestēt Car labojumus) un LiterPerKilometer vairs nebija saderīgs ar jauno Car. Nezināju, vai tādā gadījumā jādzēš LitersPerKilometer, tāpēc izkomentēju.